deps/obs-scripting/obspython/obspython.i

```diff
@@ -41,6 +41,9 @@
 
 %}
 
+%feature("python:annotations", "c");
```

Comment:
Would prefer using Python annotations or PEP 484 annotations if available.

Reply:
So only C annotations are available right now? Fair enough, we take what we can get.

Reply:
Unfortunately, yeah. @CodeYan01 and I are experimenting with potential alternatives, but honestly none of them are looking fantastic.